### ISSUE TITLE
Training model 

### ISSUE THREAD

**Issue by yiwei0730** [litagin02/Style-Bert-VITS2#80]
When I do not load the original checkpoint and want to retrain the model (because I have increased the number of languages ​​or added other module parts), **are there any techniques or tricks for training this model?** 
I saw that you use warmup to set the learning rate.
In addition, I saw a reply on the BV2 project that said "For the first.We add net_dur_disc in the latest 50k/30k steps." But I don't quite understand what this means.

**Comment by OzoneAsai** [comment]
I don't understand what you mean, but I sometimes use the pretrained model with other data to train new model as continue training
(
model_A
 `config.json`
 `model_A_e1_s800.safetensors`
 move to the following folder, and rename it.
 model_b
  `model_b_e1_s800.safetensors`
And then, continue training in model_b without generating style file.

I hope this can be your help.
(But I don't understand what do you meant that you are to retrain model without loading original checkpoint...
Sorry...)

**Comment by yiwei0730** [comment]
sorry maybe i said too hard. 
I just mean just train the model, not using the pretrained model checkpoint
and asking are ther **any tips or tricks you use for training the model** when you train another model?

**Comment by litagin02** [comment]
I haven't trained from scratch, the pretrained models are due to original [Bert-VITS2](https://github.com/fishaudio/Bert-VITS2) project, so I have no tips.

I know some Japanese train from scratch without using pretrained models, without any tricks. So just usual train and not place any pretrained model. Although lots of steps are required, it seems possible to pronounce correctly (even though pretrained model is more better).

**Comment by yiwei0730** [comment]
Thank you for your reply. I was just thinking that if a warmup epochs is set, there should be some methods set previously XD
